### PR TITLE
Bug/100/fix editor scrolling

### LINKED ISSUE #100: Fix editor scrolling
__Description:__
Currently the editor guesses the bottom of the editor section using a hard-coded value, this breaks scrolling sometimes. Additionally scrolling when in the editor scrolls both the sidebar and the webpage - this isn't what a user would expect.

__Requires:__
* editor section to function

__Breakdown:__
- [x] Fix hard-coded padding
- [x] When the cursor is over the editor panel scroll only the editor, if the cursor is over the webpage scroll only the webpage

__Time Estimation:__
2 hrs.

### PR DESCRIPTION
**Pull request by bndnio**
Resolves #100

### CODE REVIEW

**Comment by bndnio** [comment]
Fixed, take a look brewskis @jcgallegdup @hwkr @kelseyll

File: client/styles/pages/editor.less [hunk 1]
```diff
@@ -21,7 +21,12 @@
     .editor-viewbox {
       overflow-y: scroll;
       max-height: 100%;
-      padding-bottom: 5.2rem; //Todo: fix hack
+      padding-bottom: 15rem; //Todo: fix hacked hack
```
Comment (okjuan) [inline client/styles/pages/editor.less:24]: lolz #life

**Comment by okjuan** [comment]
looks good to me!

**Review by okjuan — APPROVED** [review]
Scrolling worked nicely when I ran it on my laptop.

**Comment by hwkr** [comment]
Couldn't we steal scroll with just css?

**Comment by bndnio** [comment]
@hwkr unfortunately not :(
The scroll is performed on the body while the side bar is floating, so it'll always bubble up to the body element (and JS solutions preventDefault and stopPropagation don't work either)

**Comment by kelseyll** [comment]
+1

**Pull request merged**